src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is a commonly used approach. This approach, which is also known as 'Big Bang' (and by other names), refers to the process where your data is moved in its entirety from a source system to a target system within a defined period of time. This can involve some application downtime or service degradation.
+
+Lift and Shift is a commonly used approach. However, it may not be the right approach if a strong application service continuity during the migration is required. It may be a viable method if application downtime is permitted.
+
+{{site.data.alerts.callout_info}}
+The information on this page assumes you have already reviewed the [Migration Overview](migration-overview.html).
+{{site.data.alerts.end}}
+
+## Pros and Cons
+
+On the spectrum of different data migration strategies, 'lift and shift' has the following pros and cons. The terms "lower" and "higher" are not absolute, but relative to other approaches.
+
+Pros:
+
+- Conceptually straightforward
+- Less complex: if you can afford some downtime, the overall effort will usually be lower, and the chance of errors is lower
+- Lower time start-to-finish: in general, the more downtime you can afford, the shorter the overall migration project timeframe can be
+- Lower technical risk: it does not involve running multiple systems alongside each other for an extended period of time
+- Easy to practice dry runs of import/export using testing/non-production systems
+- Good import/export tooling is available (e.g. external tools like: [AWS Database Migration Service (DMS)](aws-dms.html), [Qlik Replicate](qlik.html), [Striim](striim.html); or internal tools like [`IMPORT INTO`](import-into.html), [`COPY`](copy-from.html), [`cockroach userdata`](cockroach-userfile-upload.html))
+- If your application already has regularly scheduled maintenance windows, your customers will not encounter application downtime
+
+Cons:
+
+- Higher project risk: It either works or does not work; once you start, you have to finish or rollback
+- Application service continuity requirements must be relaxed (that is, application downtime or degradation may be needed)
+- Less control of the pace (the project *has* to be completed to meet a given downtime / service degradation window)
```

Comment:
Maybe say "All or Nothing" instead of less control of the pace since i don't think pace itself is the concern but more the all or nothing. I think i'd name the first con all or nothing and then have the current test. I'd make a new bullet that says "high project risk" which communicates that there are fewer ways to mitigate migration complications during the  process"

Reply:
sounds good - renamed the one bullet to 'all or nothing' and edited the bullets as you suggested (if I understood correctly) - thanks!